guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,206 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Your App can Listen to Events, through secure web hooks. Also, your App can handle actions that may be invoked 
```

Comment:
See my previous comment.  

https://github.com/watsonwork/watsonwork-developer-docs/pull/105#pullrequestreview-143710937

I think you removed the wrong paragraph and kept the one to remove